tpcx_bb/queries/q18/tpcx_bb_query_18_sql.py

```diff
@@ -291,6 +306,12 @@ def main(data_dir, client, bc, config):
         ORDER BY s_name, r_date, r_sentence, sentiment_word
     """
     result = bc.sql(query_4)
+
+    bc.drop_table("stores_with_regression")
```

Comment:
move this to after `temp_table2 = bc.sql(query_3)` and add `del stores_with_regression`

Reply:
same with `combined`